pkg/api/vlabs/types.go

```diff
@@ -118,6 +118,14 @@ const (
 // OrchestratorProfile contains Orchestrator properties
 type OrchestratorProfile struct {
 	OrchestratorType OrchestratorType `json:"orchestratorType"`
+	KubernetesConfig KubernetesConfig `json:"kubernetesConfig,omitempty"`
+}
+
+// KubernetesConfig contains the Kubernetes config structure, containing
+// Kubernetes specific configuration
+type KubernetesConfig struct {
```

Comment:
In another discussion on ACS Talk someone had suggested adding an array of orchestrator options instead of typed property which would make this model flexible. E.g.:
“orchestratorOptions”: [
         “KubernetesImageBase”: "xyz",
         “KubectlVersion”: “1.5.2”
         ]

I think this is a good proposal. @wangtt03, @colemickens, @anhowe: what do you think?